2024/fail_fast/employee_portal/after_main.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import FastAPI, HTTPException
+from decimal import Decimal
+from typing import Any, List, Tuple
+from employee_portal.database.database import SQLiteConnector
+from employee_portal.employees import create, delete, index, show, update
+from employee_portal.payment import (
+    CreditCardPayment,
+    PaymentProcessor,
+    RedeemBuddyPayment,
+)
+from employee_portal.position import Position
+
+app = FastAPI()
+
+connector = SQLiteConnector()
+
+credit_card_payment = CreditCardPayment()
```

Comment:
If these are globals, they should be uppercase. An alternative way of dealing with this is by using FastAPI's built-in dependency injection framework.